File: DPBO-praktikum/2/CPP/Human.cpp
#include <iostream>
#include <string>
using namespace std;

class Human {
    protected:
        string NIK;
        string nama;
        string gender;

    public:
        // Konstruktor
        Human(string NIK, string nama, string gender) {
            this->NIK = NIK;
            this->nama = nama;
            this->gender = gender;
        }

        // Getter untuk NIK
        string getNIK() {
            return NIK;
        }
        void setNIK(string NIK) {
            this->NIK = NIK;
        }
        
        // Getter untuk nama
        string getnama() {
            return nama;
        }
        void setNama(string nama) {
            this->nama = nama;
        }

        // Getter untuk jenis kelamin
        string getGender() {
            return gender;
        }
        void setGender(string gender) {
            this->gender = gender;
        }
};

File: DPBO-praktikum/2/Java/Human.java
public class Human {
    protected String NIK;
    protected String nama;
    protected String gender;

    // Konstruktor
    public Human(String NIK, String nama, String gender) {
        this.NIK = NIK;
        this.nama = nama;
        this.gender = gender;
    }

    // Getter untuk NIK
    public String getNIK() {
        return NIK;
    }
    public String setNIK(String NIK) {
        return this.NIK=NIK;
    }
    
    // Getter untuk nama
    public String getnama() {
        return nama;
    }
    public String setNama(String nama) {
        return this.nama=nama;
    }

    // Getter untuk jenis kelamin
    public String getGender() {
        return gender;
    }
    public String setGender(String gender) {
        return this.gender=gender;
    }
}
